output/slide-deck.qmd

```diff
@@ -238,9 +255,10 @@ will see later on what these files are)
 <br>
 
 ::: nonincremental
-1.  **In your GitHub repo** click on the "Code" button
```

Comment:
This was not 100% clear to me, since we had potentially moved to a different tab

Reply:
Because I think before this slide they end up adding a collaborator on the Settings tab, so won't be on the Code tab to begin with